### PR TITLE
Modifications to script and manual

### PR DESCRIPTION
**Pull request by carlos-encs**
Modifications: --mem to scripts, Manual:jupyter, vEnvironments
To do: generate the web manual

### CODE REVIEW

File: doc/speed-manual.tex [hunk 1]
```diff
@@ -340,7 +340,7 @@ \section{Job Management}
 Job instructions in a script start with \verb+#SBATCH+ prefix, for example:
 \begin{verbatim}
 #SBATCH --account=speed1 --mem=100M -t 600 -J job-name
-#SBATCH --gpus=2 --mail-type=ALL -t 600 --mail-user=$USER
+#SBATCH --gpus=2 --mem=1Gb --mail-type=ALL -t 600 --mail-user=$USER
```
Comment (smokhov) [inline doc/speed-manual.tex:343]: We already have `--mem` on line 342, so the one on 343 is not needed.

File: doc/scheduler-scripting.tex [hunk 2]
```diff
@@ -561,6 +561,55 @@ \subsubsection{Graphical Applications}
 	\label{fig:pycharm}
 \end{figure}
 
+This is an example of running Jupyter-Notebooks.
```
Comment (smokhov) [inline doc/scheduler-scripting.tex:564]: We need a `\subsubsection{Jupyter Notebooks}` before this line, so it shows up in the TOC, and easily linkable in tickets and replies.

File: src/array.sh [hunk 3]
```diff
@@ -4,6 +4,7 @@
 #SBATCH -c 1
 #SBATCH -N 1
 #SBATCH -t 0-2:00
+#SBATCH --mem=4Gb
```
Comment (smokhov) [inline src/array.sh:7]: Duplicate `--mem`, see the one a couple of lines below.

File: doc/images/jupyter.PNG
Comment (smokhov) [inline doc/images/jupyter.PNG:1]: Please lowercase `.png` extension in the filename; some systems may flag it as an error when building the manual. Can use `git mv jupyter.PNG jupyter.png`. The same applies to the other PNG files.

File: doc/scheduler-scripting.ps
Comment (smokhov) [inline doc/scheduler-scripting.ps:1]: Please remove this file, it does not belong in the repo.

File: doc/speed-manual.pdf
Comment (smokhov) [inline doc/speed-manual.pdf:1]: Please only add PDF at the very end usually close to a release, when we've accumulated enough changes. Please don't routinely commit it together with .tex files.

**Review by smokhov — CHANGES_REQUESTED** [review]
We are also missing an emphasized note that `--mem` is now required else job submission will fail.

File: src/array.sh (hunk 3 above)
Comment (carlos-encs) [inline src/array.sh:7]: fixed

**Comment by smokhov** [comment]
Looks good.

**Review by smokhov — APPROVED** [review]

**Pull request merged**